element-connector/src/test/java/org/eclipse/californium/elements/util/NamedThreadFactoryTest.java

```diff
@@ -0,0 +1,87 @@
+
+/*******************************************************************************
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ *
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ *
+ * Contributors:
+ *    Achim Kraus (Bosch Software Innovations GmbH) - initial creation
+ ******************************************************************************/
+
+package org.eclipse.californium.elements.util;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import java.util.concurrent.CountDownLatch;
+import org.junit.Test;
+
+public class NamedThreadFactoryTest {
+
+	@Test
+	public void testDaemonThreadGroup() {
+		ThreadGroup group = new ThreadGroup("test-group");
+		group.setDaemon(true);
+		createThread(group);
+		assertTrue("group is not destroyed", group.isDestroyed());
+	}
+
+	@Test
+	public void testThreadGroup() {
+		ThreadGroup group = new ThreadGroup("test-group");
+		group.setDaemon(false);
+		createThread(group);
+		assertFalse("group is destroyed", group.isDestroyed());
+	}
+
+	@Test
+	public void testScandiumThreadGroup() {
+		ThreadGroup group = new ThreadGroup("test-group");
```

Comment:
:thinking:  I suspect there is a copy paste issue because I don't see any difference between : 
testThreadGroup
testScandiumThreadGroup
testCaliforniumThreadGroup

Reply:
Yes, copy & paste. Fixed.